src/api/routes/cron.ts

```diff
@@ -39,6 +42,28 @@ const cron = router({
     });
     await Promise.all(promises);
   }),
+
+  /**
+   * Check the numbers of meetings in progess by each Tencent user id
```

Comment:
The comment is confusing. We don't have multiple tencent users yet. so no need to mention "each". We should update the comment when multi-user support is added.